data/card_catalogue/card-manipulation.json

```diff
@@ -116,19 +116,6 @@
       "image": "res/art/illustration/Copy.png"
     }
   },
-  {
-    "title": "Master Mimic",
-    "upgrade": null,
-    "tier": "red",
-    "details": {
-      "rules": {
-        "active": "Copy the effect of the last red cast by any character.",
-        "passive": null
-      },
-      "flavour": "“Through others we become ourselves.” ― Mad Dog Millson, Revered Pirate and Thief",
-      "image": "res/art/illustration/Master Mimic.png"
-    }
-  },
```

Comment:
This card is incredibly strong in the right situation, but doesn't really have a fantasy of its own. It's also a terrible whiff in the early game, since it's quite likely that nobody has ever cast a red card before when this fires.